src/apps/benchmarks/benchmarkGridDiskCells.c

```diff
@@ -23,7 +23,9 @@ H3Index pentagon = 0x89080000003ffff;
 
 BEGIN_BENCHMARKS();
 
-H3Index *out = malloc(H3_EXPORT(maxGridDiskSize)(40) * sizeof(H3Index));
+int64_t outSz;
+H3_EXPORT(maxGridDiskSize)(40, &outSz);
```

Comment:
Should the benchmarks check for the error condition?

Reply:
Updated for the setup part. I didn't feel the benchmark should check this as part of what it times since this isn't part of the code we're intending to measure.